schema/deploy/shipping/views.sql

```diff
@@ -3694,6 +3694,69 @@ grant select
   on shipping.uw_reopening_encounters_v1
   to "incidence-modeler";
 
+
+create or replace view shipping.uw_reopening_encounters_hct_data_pulls as
+(
+  with enrollments as
+  (
+  select encounter_id as enrollment_encounter_id
+  , individual_id as enrollment_individual_id
+  from warehouse.encounter
+  where
+      encounter.identifier like 'https://hct.redcap.rit.uw.edu/148/%/enrollment_arm_1/'
+  ),
+
+  encounters as
+  (
+  select encounter_id
+  , encountered
+  , age
+  , individual_id
+  , site_id
+  , details
+  from warehouse.encounter
+  where
+      encounter.identifier like 'https://hct.redcap.rit.uw.edu/148/%/encounter_arm_1/%'
+  )
+
+  select
+  encounter.encounter_id
+  , encounter.encountered
+  , encounter.age
+
+  , individual.sex
+
+  , enroll_details.affiliation
+  , enroll_details.student_level
+  , enroll_details.athlete
+  , enroll_details.core_race
+  , enroll_details.core_latinx
+  , enroll_details.campus_location
+  , enroll_details.core_house_members
+
+  from encounters encounter
+  join enrollments enroll on encounter.individual_id = enroll.enrollment_individual_id
+  join warehouse.individual individual on individual.individual_id = encounter.individual_id
+  join shipping.uw_reopening_enrollment_fhir_encounter_details_v1 enroll_details on enroll_details.encounter_id = enroll.enrollment_encounter_id
+
+  -- Filter out follow up encounters
+  where not encounter.details @> '{"reason": [{"system": "http://snomed.info/sct", "code": "390906007"}]}'
```

Comment:
Do you mind explaining a bit what youre intending here? In this view, are you trying to have one row per individual or per any encounter or certain types of encounters? Most of the variables seem to be coming from enrollments but since youre merging to encounters youre ending up with multiple encounter records per individual in the view.

Reply:
I see your point now. I guess I want one row per individual for the enrollment data so I don't duplicate any, but then I do still want one row per encounter so that I can pull in all encounter data for aggregation.